pgx-pg-config/src/lib.rs

```diff
@@ -138,9 +138,9 @@ impl PgConfig {
     ///
     /// It also requires that the `PGX_PG_CONFIG_AS_ENV` variable be set to some value that isn't
     /// the string `"false"`.
-    pub fn from_env() -> Option<Self> {
+    pub fn from_env() -> eyre::Result<Self> {
```

Comment:
Is there still a reason to make this return a Result?

Reply:
Is asking for it from the environment when it’s not there akin to asking for it from a file that doesn’t exist?  And would we return a Result in that case?

obviously I could go either way as v1 of this was an Option.

Reply:
I'm gonna stick with Result